src/bidiMapper/domains/context/browsingContextImpl.ts

```diff
@@ -180,8 +180,8 @@ export class BrowsingContextImpl {
     this.#children.set(child.contextId, child);
   }
 
-  async #deleteChildren() {
-    await Promise.all(this.children.map((child) => child.delete()));
+  #deleteChildren() {
```

Comment:
delete() is not `async`, I wonder why there was an `await` here in the first place?

I was going to suggest to add a comment here to state that this is purposely without `await`, but there's no need because the underlying function isn't async anyway.

Reply:
Actually, disregard this, I reviewed an older commit. Let me take another look.